R/data.table.R

```diff
@@ -2770,14 +2770,6 @@ address = function(x) .Call(Caddress, eval(substitute(x), parent.frame()))
   stopf('Check that is.data.table(DT) == TRUE. Otherwise, :=, `:=`(...) and let(...) are defined for use in j, once only and in particular ways. See help(":=").')
 }
 
-J = function(...) {
```

Comment:
I think we can leave these here as long as they're not exported

Reply:
done 👍

Reply:
I don't know how this passed, but test 715 is now failing locally on my system and I think it should have failed in our CI too.